src/inventory.cc

```diff
@@ -5647,7 +5617,7 @@ static int inventoryQuantitySelect(int inventoryWindowType, Object* item, int ma
             }
 
             soundPlayFile("iisxxxx1");
-        } else if (keyCode == 5000) {
+        } else if (keyCode == 5000 || keyCode == KEY_LOWERCASE_A) {
```

Comment:
Use `a` to click `All` in item quantity modal.  Doesn't close the modal (but could, perhaps…)

Reply:
This is neat. Although sfall does this a little different. IIRC shift+drag move the whole stack without the need to even open the quantity screen.

Reply:
This PR also has Ctrl-click to move the whole stack without opening the quantity screen.  To me it doesn't seem necessary that they are independent shortcuts.  If you want to use the quantity picker, dragging doesn't seem that onerous.

(And `a` is useful in that context)